packages/mobile/src/account/SupportContact.tsx

```diff
@@ -59,7 +59,7 @@ function SupportContact({ route }: Props) {
       network: DEFAULT_TESTNET,
     }
     const userId = e164PhoneNumber ? anonymizedPhone(e164PhoneNumber) : 'unknown'
-    const emailSubject = 'Celo support for ' + (userId || 'unknownUser')
+    const emailSubject = 'Valora support for ' + (userId || 'unknownUser')
```

Comment:
Proposing to use the existing constant we have:
```suggestion
    const emailSubject = `${APP_NAME} support for ${userId || 'unknownUser'}`
```

Reply:
By the way, we should translate this.

Reply:
I think the emails are sent to our support team, which is why it's just in English, do you think it needs to be translated to user's language?

Reply:
Yes please.